Userland/Libraries/LibGUI/EmojiInputDialog.h

```diff
@@ -28,9 +28,15 @@ class EmojiInputDialog final : public Dialog {
 private:
     explicit EmojiInputDialog(Window* parent_window);
 
+    struct EmojiScore {
+        Emoji* emoji;
```

Comment:
Can be a reference.

Reply:
I already pointed that out earlier.
https://discord.com/channels/830522505605283862/830739873119207426/1268611093232750703